docs: fix exclude option for quickstart examples. (#5854)
dagger/dagger: docs/current/quickstart/947391-test.mdx

- It creates a Dagger client with `Connect()` as before.
- It uses the client's `Container().From()` method to initialize a new container from a base image - again, the `node:16-slim` image. This base image is the Node.js version to use for testing. The `From()` method returns a new `Container` object with the result.
- It uses the `Container.WithDirectory()` method to mount the source code directory on the host at the `/src` mount point in the container, and the `Container.WithWorkdir()` method to set the working directory to that mount point.

>   - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and build (build artifacts) irectories.

Reviewer: Is `d` in `irectories` is missing?

```suggestion
  - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and build (build artifacts) directories.
```

Reviewer 2: @marcosnils this text should also be adjusted for the other two tabs in this page.

Reviewer 2: @marcosnils I think you may have missed my comment above

>   - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and build (build artifacts) irectories.

Reviewer 2: ```suggestion
  - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

</TabItem>
<TabItem value="Node.js">

<Embed id="Wb_vszr9Ule" />

This code listing does the following:

- It creates a Dagger client with `connect()` as before.
- It uses the client's `container().from()` method to initialize a new container from a base image - again, the `node:16-slim` image. This base image is the Node.js version to use for testing. The `from()` method returns a new `Container` object with the result.
- It uses the `Container.withDirectory()` method to mount the source code directory on the host at the `/src` mount point in the container, and the `Container.withWorkdir()` method to set the working directory to that mount point.

>   - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.

Reviewer 3: ```suggestion
  - Notice that the `Container.withDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

Reviewer 2: Thank you! https://github.com/dagger/dagger/pull/6142

</TabItem>
<TabItem value="Python">

<Embed id="yZj9P2N03-D" />

This code listing does the following:

- It creates a Dagger client with `with dagger.Connection()` as before.
- It uses the client's `container().from_()` method to initialize a new container from a base image - again, the `node:16-slim` image. This base image is the Node.js version to use for testing. The `from_()` method returns a new `Container` object with the result.
- It uses the `Container.with_directory()` method to mount the source code directory on the host at the `/src` mount point in the container, and the `Container.with_workdir()` method to set the working directory to that mount point.

>   - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.

Reviewer 3: ```suggestion
  - Notice that the `Container.with_directory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

Reviewer 2: Thank you! https://github.com/dagger/dagger/pull/6142
